sorbet/rbi/gems/actionpack.rbi

```diff
@@ -1765,6 +1765,10 @@ class ActionDispatch::RemoteIp::GetIp
   def ips_from(header); end
   def to_s; end
 end
+module ActionController::ApiRendering
```

Comment:
I believe all the sorbet stuff was generated because I rebased on `main` with dep updates. I think it's safe to ignore.